apps/notifier/src/pull-listener.ts

```diff
@@ -15,14 +21,81 @@ export default async function startPullListener(): Promise<void> {
     const subscription = await pubsubClient.getSubscription(pubsubSettings.topicName, pubsubSettings.subscriptionName)
     logger.info('listening messages from topics')
 
+    sgMail.setApiKey(SENDGRID_APY_KEY);
+
     subscription.on('message', async (message: Message) => {
       const { attributes } = message
+      const { topic } = attributes;
+      const dataJson = JSON.parse(message.data.toString());
+
+      const pathToAttachment = `${__dirname}/src/assets/views/hola.html`;
+      //const attachment = fs.readFileSync(pathToAttachment,'utf8');
+      console.log("antes del template")
+      nunjucks.configure('views',{autoescape: true});
+      console.log("en medio del template") 
+      // D:\Escritorio\Master\Fundamentos de cloud\bike4life\apps\notifier\src\views\base.html
+      // D:\Escritorio\Master\Fundamentos de cloud\bike4life\apps\notifier\src\pull-listener.ts
+
+      const Nuntemplate = nunjucks.render('/views\/base.html',dataJson);
+      console.log("despues del template")
+    
+      
+
+      if(topic === "UserCorrectlyCreated") {
```

Comment:
The topic will never change, as there will be a topic called something like `NotifierMessagesTopic` where all messages for this component will be published. We should check the messages attributes, as there will be an attribute called `type` where we'll send the proper type for each message.